omnibus/files/private-chef-cookbooks/private-chef/resources/chef_run.rb

```diff
@@ -1,74 +0,0 @@
-#
```

Comment:
Removing this is the big win as it represents a source of pain on every major chef-client upgrade.